src/gei/Commands/MigrateRepo/MigrateRepoCommandHandler.cs

```diff
@@ -64,7 +64,7 @@ public async Task Handle(MigrateRepoCommandArgs args)
 
         _log.LogInformation("Migrating Repo...");
 
-        var blobCredentialsRequired = await _ghesVersionChecker.AreBlobCredentialsRequired(args.GhesApiUrl, args.UseGithubStorage);
+        var blobCredentialsRequired = !args.UseGithubStorage || (await _ghesVersionChecker.AreBlobCredentialsRequired(args.GhesApiUrl));
```

Comment:
As described [here](https://github.com/github/gh-gei/pull/1293#discussion_r1838610636) we do need to always check the GHES version to be able to log the warnings.

Reply:
so this is what I mean:

```suggestion
        var blobCredentialsRequired = await _ghesVersionChecker.AreBlobCredentialsRequired(args.GhesApiUrl);
```